recipe/meta.yaml

```diff
@@ -23,12 +23,16 @@ requirements:
   run:
     - python
     - absl-py >=0.1.6
+    - astor >=0.6.0
     - enum34 >=1.1.6  # [py<34]
+    - gast >=0.2.0
+    - grpcio >=1.8.6
     - mock >=2.0.0    # [py27]
-    - numpy >=1.12.1
+    - numpy >=1.13.3
     - protobuf >=3.4.0
     - six >=1.10.0
-    - tensorboard >=1.5.0,<1.6.0
+    - tensorboard >=1.6.0,<1.7.0
+    - termcolor >=1.1.0
```

Comment:
Strangely even the Windows wheels require `termcolor`. Though they don't require `colorama` on Windows, which is a bit unexpected. Not totally sure how this is suppose to work correctly on Windows.